integrated_channels/blackboard/admin/__init__.py

```diff
@@ -38,3 +39,19 @@ def enterprise_customer_name(self, obj):
                 being rendered with this admin form.
         """
         return obj.enterprise_customer.name
+
+    def oauth_authorization_url(self, obj):
+        """
+        Returns: the oauth authorization url when the blackboard_base_url and client_id are available.
+
+        Args:
+            obj: The instance of BlackboardEnterpriseCustomerConfiguration
+                being rendered with this admin form.
+        """
+        if obj.blackboard_base_url and obj.client_id:
+            return (f"{obj.blackboard_base_url}/learn/api/public/v1/oauth2/authorizationcode"
```

Comment:
I think this base url also needs to be parameterized? `https://courses.edx.org`. Meaning we may also have staging configurations, or local. Figure this is best obtained from a LMS_BASE_URL settings somewhere?

Reply:
i believe the `blackboard_base_url` is user supplied as part of the configuration - it points to their instance.

Reply:
oh sorry I meant in the redirect_uri

Reply:
would it be `settings.LMS_BASE_URL/blackboard/oauth-complete' ?

Reply:
oh i see what you mean - of course. let me take a look.